src/subpages/newsSubpage/NewsData.js

```diff
@@ -24,18 +32,16 @@ export default class NewsSubpage extends Subpage {
   }
 
   async updatePage() {
-    const newsRes = await this._newsApi.getWorldNews()
+    const worldNews = await this._newsApi.getWorldNews()
+    const polandNews = await this._newsApi.getPolandNews()
+    const healthNews = await this._newsApi.getHealthNews()
 
     let outputMessage = ""
-    newsRes.articles.forEach((e) => {
-      outputMessage += this.generateMessage(
-        e.title,
-        e.author,
-        e.content.substring(0, 200)
-      )
-
-      this.getNewsContentDiv().innerHTML = outputMessage
-    })
+    outputMessage += this.generateMessage(outputMessage, worldNews)
+    outputMessage += this.generateMessage(outputMessage, polandNews)
+    outputMessage += this.generateMessage(outputMessage, healthNews)
```

Comment:
```suggestion
    let output message = ""
    await this._newsApi.getWorldNews().then(res => outputMessage += this.generateMessage(outputMessage, worldNews))
```
i tak z każdą z tych trzech kategorii newsów możesz zrobić